### PR TITLE
Removed superfluous python-xlib dependency

### PR DESCRIPTION
**Pull request by gibbz00**
Addresses #198. 

This was a feature released in 1.3.0 (see changelog), but then regressed by a merge request in 2.0.1, #116 

Someone must have realized that this replacement method was a bad idea, because the current branch is back to using the `i3/sway --get-socketpath` method. This revert was however incomplete, as the method added to #116 still remains, along with the `python-xlib` dependency. 

Anyhow, the pull request purges any use of `python-xlib`. It's still used in testing, but there the package is separately installed when creating the docker testing image. 

TL;DR: Don't accept pull request by script kiddies that address non-existent issues, regress features, and add package dependencies; all in the name of "removing a subprocess call" by replacing it with 5 method calls to `python-xlib`.

### CODE REVIEW

**Comment by acrisci** [comment]
Ok it's probably better this way.
